getting-started.md

```diff
@@ -43,7 +43,9 @@ To run your app from the command-line:
 
 1. Open a terminal and change directories to the root of your app (the same directory that
 contains the `pubspec.yaml` file for your project).
-2. Run the following command.
+2. Run the following command. Once the `flutter` tool is running, if you change the app's
```

Comment:
We might want to split this into a third step. After the app is running, we can ask the user to open any text editor, change a specific thing, then hit 'r'.